frontend/src/components/4.Preferences/Preferences.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+import {useState} from 'react';
+import Help from '../Common/Help';
+import '../Common/Help.css';
+import styled from 'styled-components';
+
+const UserInput = styled.input`
+  outline: none;
+  width: 80vw;
+  border-radius: 20px;
+  border: 4px solid #000000;
+  font-size: 4vw;
+  margin: 1vw;
+`;
```

Comment:
See prior comments - this style block seems similar to that in JoinGroup and could probably be pulled out

Reply:
i think the intention was for the person working on the page to refactor it out to the correct style, since we wanted a quickly template.

Reply:
Okay, that rationale is understandable but may end up leaving the in-class styles there semi-permanently. 

How difficult would it be to split it out into its own stylesheet?

Reply:
i dont think it will be too hard. It might just cause us to spend more time until the rest of us can start implementation

Reply:
Seeing as the Preferences block and the JoinGroup blocks are nearly identical, I think I'd prefer if both could be pulled out, as we may start running into code duplication issues